src/test/java/ProjectServiceTest.java

```diff
@@ -47,4 +47,11 @@ public void createTest(){
             assertNotNull(attribute.getId());
         }
     }
+
+    @Test
+    public void checkSimilarNamesCreationTest(){
+        Project resultProject = projectService.createProject(testProject);
+        Project resultProject2 = projectService.createProject(testProject);
```

Comment:
Second calling of projectService.createProject(testProject) will failed and exception will throw. It is correct. Your test should check that particular exception is really throw.